views/apps/new.html

```diff
@@ -17,19 +17,44 @@ <h2 class="font-bold text-2xl font-headline mb-2">Add a new session</h2>
     </p>
   </div>
 
-
-  <div class="mb-4">
+  <div class="mb-6">
     <h3 class="text-xl font-headline mb-2">Permissions</h3>
-    <p class="text-sm mb-4">Grant this application session access to your wallet to:</p>
+    <p class="text-sm mb-4">Grant this application access to your wallet to:</p>
     <ul>
       <li class="mb-2 relative pl-6">
         <i class="fas fa-check mr-3 text-green-700 absolute left-0 top-1" aria-hidden="true"></i>
-        Send payments from your account.
+        <p>
+          Send payments from your account.
+        </p>
+        <p class="text-gray-500 dark:text-gray-400">
+          <input type="checkbox" class="text-gray-500 dark:text-gray-400"
+            onclick="document.getElementById('budget').classList.toggle('hidden');">
+          Set a budget
+        </p>
+        <div id="budget" class="hidden my-4">
+          <div class="mt-2">
+            <label for="MaxAmount" class="block mb-2 text-sm font-medium text-gray-900 dark:text-white">
+              Monthly amount in sats
```

Comment:
I think MaxAmount is a bit confusing if it is actually a monthly max amount and not a total max amount. Maybe this can be improved later on though, if it is also possible to specify the budget interval

Reply:
I think this is OK combined with BudgetRenewal